src/style.css

```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+html {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+  font-size: 300;
+}
+
+.search-bar {
+  box-sizing: border-box;
```

Comment:
it is common practice to apply `box-sizing: border-box` for all elements using `*` selector